ehrapy/api/preprocessing/encoding/__init__.py

```diff
@@ -44,35 +46,40 @@ def encode(
             adata = ep.io.read(...)
             # encode col1 and col2 using label encoding and encode col3 using one hot encoding
             adata_encoded = ep.encode.encode(adata, autodetect=False, {'label_encoding': ['col1', 'col2'], 'one_hot_encoding': ['col3']})
+
+        Another example with multiple :class:`~anndata.AnnData` objects.
+
+        .. code-block:: python
+
+            import ehrapy.api as ep
+            adata_objects = ep.io.read(...)
+            # encode col1 and col2 of adata1 using label encoding and encode col1 of adata2 using one hot encoding
+            adata_encoded = ep.encode.encode(adata_objects, autodetect=False, {'adata1': {'label_encoding': ['col1', 'col2']},
+            'adata2': {'one_hot_encoding': ['col3']}})
     """
     return Encoder.encode(data, autodetect, encodings)
 
 
-def undo_encoding(
-    adata: AnnData, columns: str = "all", from_cache_file: bool = False, cache_file: str = None
-) -> AnnData:
+def undo_encoding(adata: AnnData, columns: str = "all") -> AnnData:
     """Undo the current encodings applied to all columns in X.
 
     This currently resets the AnnData object to its initial state.
     Args:
         adata: The :class:`~anndata.AnnData` object
         columns: The names of the columns to reset encoding for. Defaults to all columns.
-        from_cache_file: Whether to reset all encodings by reading from a cached .h5ad file, if available.
-        This resets the :class:`~anndata.AnnData object to its initial state.
-        cache_file: The filename of the cache file to read from
 
     Returns:
-        A (partially) encoding reset :class:`~anndata.AnnData` object
+        A reset, unencoded :class:`~anndata.AnnData` object
 
     Example:
        .. code-block:: python
 
            import ehrapy.api as ep
            # adata_encoded is a encoded AnnData object
-           adata_undone = ep.encode.undo_encoding(adata_encoded)
-           # adata_undone is a fully reset AnnData object with no encodings
+           adata_reset = ep.encode.undo_encoding(adata_encoded)
+           # adata_reset is a fully reset AnnData object with no encodings
```

Comment:
Usually not a fan of comments below a code of line. Either same line or above. What do you think? I'd try to go for the same line here if it's not crazy long.